framework/db/Connection.php

```diff
@@ -618,9 +624,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = Yii::createObject([
```

Comment:
It will allow dependency injection in this way. Original issue was about such ability.